samples/Gallery/Pages/NotificationsPage.fs

```diff
@@ -50,6 +81,8 @@ module NotificationsPage =
         match msg with
         | ShowManagedNotification ->
 
+            //TODO this changes global state! i.e. after receiving this message, all notifications appear bottom right. is that intended?
```

Comment:
Bear in mind that all of this is a port for the Avalonia Catalog samples, so I just wanted to test in the control works for us. have not really put too much though on making this `side-effect` free per se. Just adding some samples

Reply:
I see. This was just something where the behaviour surprised me. Not so much after seeing the assignment `NotificationManager = FabApplication.Current.WindowNotificationManager` in the source - but it still raised the question of how I'd customize the position of just one message without doing the same for all following messages. Would I have to remember the original Position and reset it after `model.NotificationManager.Show`?